docs/topics/proc_starting-your-application-locally-in-debugging-mode.adoc

```diff
@@ -17,15 +26,36 @@ This is useful especially when performing integration testing.
 .Procedure
 
 . In a console, navigate to the directory with your application.
+ifdef::wf-swarm[]
 . Launch your application and specify the debug port using the `-Dswarm.debug.port` argument:
+endif::[]
+ifdef::vertx[]
+. Launch your application and specify the debug port using the `-Ddebug.port` argument:
+endif::[]
+ifdef::spring-boot[]
+. Launch your application and specify the debug port using the `-Dspring-boot.debug.port` argument:
```

Comment:
Did you verify that these system properties are actually supported by the respective runtimes?

Reply:
Please ignore, addressed in #1005.